tracer/test/Datadog.Trace.Tests/Debugger/SymbolsTests/SymbolExtractorTest.cs

```diff
@@ -162,16 +183,21 @@ private VerifySettings ConfigureVerifySettings(string className)
         return settings;
     }
 
-    private Root GetSymbols(Assembly assembly, string className)
+    private (Root Symbols, Exception Error) GetSymbols(Assembly assembly, string className)
     {
         var root = GetRoot();
         var extractor = SymbolExtractor.Create(assembly);
         Assert.NotNull(extractor);
+        if (extractor is not SymbolPdbExtractor)
+        {
+            return (null, new NoPdbException());
+        }
```

Comment:
You could just throw the `NoPdbException` here I think? Then you can can remove the extra handling in the methods, and change `[SkippableTheory]` to `SkippableTheory(typeof(NoPdbException), typeof(SkipException)]` which might be more obvious?

Reply:
Fixed in dedff7b73d5a6bcbe95a2792c3079e4fe9f32b0a